core/src/com/mygdx/game/config/TechnologyConfig.java

```diff
@@ -0,0 +1,21 @@
+package com.mygdx.game.config;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class TechnologyConfig implements EntityConfig, TextureConfig {
```

Comment:
W sumie to miała być tylko wizualizacja teraz, co nie? Więc nie ma jeszcze tutaj planów na dopasowanie technologii działania?

Reply:
Nie ma narazie